virt/virt-4-10-release-notes.adoc

```diff
@@ -117,20 +122,18 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-10-bug-fixes"]
 == Bug fixes
 
+* If you hot-plug a virtual disk and then force delete the `virt-launcher` pod, you might lose data. This is due to a race condition that can cause the VM disk's contents to be wiped from the persistent volume. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2007397[*BZ#2007397*])
```

Comment:
Because these are bug fixes, my suggestion would be to tweak the wording to say how these issues are resolved. For example, you could say "If you hot-plug a virtual disk and then force delete the `virt-launcher` pod, you no longer lose data", or whatever is appropriate for that bug.

Applies to both bug fixes. Refer to 4.9 bug fixes for more examples.

Reply:
Changed these to:

* If you hot-plug a virtual disk and then force delete the `virt-launcher` pod, you no longer lose data. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2007397[*BZ#2007397*])

* In previous releases, if a cloning operation was initiated before the clone source became available, the cloning operation stalled. This is no longer an issue, as the clone authorization does not expire before the cloning operation starts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1855182[*BZ#1855182*])